packages/web/src/components/Icon/Icon.tsx

```diff
@@ -1,12 +1,52 @@
 import React from 'react'
 
+import { ColorValue, toCSSVariableName } from '@audius/stems'
```

Comment:
I added the size and color implementation to this wrapper component because I was getting tired of setting styles individually per-instance of icon that I was using. 😁

Reply:
cc @dylanjeffers